x-pack/filebeat/docs/inputs/input-cometd.asciidoc

```diff
@@ -0,0 +1,65 @@
+[role="xpack"]
+
+:type: cometd
+
+[id="{beatname_lc}-input-{type}"]
+=== CometD input
+
+++++
+<titleabbrev>CometD</titleabbrev>
+++++
+
+Use the `cometd` input to stream the real-time events from a Salesforce generic subscription Push Topic.

```

Comment:
Are there links that can be added pointing to reference information for the cometd and the salesforce interface. Are these relevant?

https://docs.cometd.org/

https://resources.docs.salesforce.com/sfdc/pdf/api_streaming.pdf

Reply:
Yes, this are relevant links. Let me add those. Thanks